services/osparc-gateway-server/src/osparc_gateway_server/backend/osparc.py

```diff
@@ -65,7 +65,7 @@ async def do_setup(self) -> None:
         self.settings = AppSettings()
         assert isinstance(self.log, logging.Logger)  # nosec
         self.log.info(
-            "osparc-gateway-server application settings:\n%s",
+            "simcore-service-gateway-server application settings:\n%s",
```

Comment:
Ok sorry I did not check that before.
The osparc-gateway-server is meant to run **outside** of simcore (and actually does not run in the simcore stack), nevertheless it needs to be synchronized which is why it is in the repo.
But, that means, that renaming is not wanted here.